playbooks/roles/dcnm_vrf/dcnm_tests.yaml

```diff
@@ -0,0 +1,39 @@
+---
+# This playbook can be used to execute integration tests for
+# the roles located in:
+#
+# REPO_ROOT/tests/integration/targets/dcnm_vrf/tests/dcnm/*.yaml
+#
+# Either:
+# 1. Modify the following:
+#    - The vars section below with details for your testing setup.
+#    - dcnm_hosts.yaml in this directory
+#
+# 2. Run the tests
+#    ansible-playbook dcnm_tests.yaml -i dcnm_hosts.yaml
+#
+# OR:
+# 1. Modify ../dynamic_inventory.py to align with your setup
+#    This must contain the vars mentioned below and controller
+#    info from dcnm_hosts.yaml (modified for your setup)
+#  2. Run the tests
+#     ansible-playbook dcnm_tests.yaml -i ../files/dynamic_inventory.py
+#
+# NOTES:
+- hosts: dcnm
+  gather_facts: no
+  connection: ansible.netcommon.httpapi
+# Uncomment and modify if not using dynamic_inventory.py
+#   vars:
```

Comment:
Is this the full set of vars that are required by the vrf integration tests?

Reply:
Good catch! I'm update it with the remaining vars.